lambda/app/src/keycloak/client.ts

```diff
@@ -12,3 +13,35 @@ export const fetchClient = async (data: {
   const { realm, client } = await getClient(kcAdminClient, { serviceType, realmName, clientId });
   return client;
 };
+
+export const disableClient = async (data: {
```

Comment:
Does this need to be exported? Are you planning on it being called elsewhere?

Reply:
hm.. I didn't see any concerns it as a public function; let's keep it as a private function for now; updated.